modules/container/src/main/java/org/mule/runtime/container/internal/DefaultPreFilteredContainerClassLoaderCreator.java

```diff
@@ -63,6 +66,11 @@ public Set<String> getBootPackages() {
     return finalBootPackages;
   }
 
+  @Override
+  public Set<String> getAdditionallyExportedResourceDirectories() {
```

Comment:
Here is used `additionally` instead of `additional` as in the variable names, it'd be better to unify it.